internal/runners/clean/cache.go

```diff
@@ -53,6 +50,7 @@ func (c *Cache) Run(params *CacheParams) error {
 				return err
 			}
 		}
+		return nil
```

Comment:
This is a different issue from the original error. Without this return, after removing a project cache it will prompt the user to remove the entire cache.